gbus/metrics/saga_metrics.go

```diff
@@ -6,8 +6,22 @@ import (
 	io_prometheus_client "github.com/prometheus/client_model/go"
 )
 
-//SagaTimeoutCounter is the prometheus counter counting timed out saga instances
-var SagaTimeoutCounter = newSagaTimeoutCounter()
+var (
+	//SagaTimeoutCounter is the prometheus counter counting timed out saga instances
+	SagaTimeoutCounter = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: grabbitPrefix,
+		Subsystem: "saga",
+		Name:      "timedout_sagas",
+		Help:      "counting the number of timedout saga instances",
+	})
+	//SagaLatencySummary is the prometheus summary for the total duration of a saga
+	SagaLatencySummary = promauto.NewSummaryVec(prometheus.SummaryOpts{
```

Comment:
Latency seems like the wrong wording for this, I would go with something like lifecycle or lifespan